Tests/Common/Data/Custom/SmartInsiderTests.cs

```diff
@@ -133,19 +134,36 @@ public void SerializeRoundTripSmartInsiderTransaction()
             Assert.AreEqual(time, deserialized.Time);
             Assert.AreEqual(time, deserialized.EndTime);
         }
-        
+
         [Test]
         public void ParseFromRawDataUnexpectedEventType()
         {
-            var rawIntentionLine = "BI12363\tSome unexpected entry\t2020-07-27\t2009-11-11\tUS1234567890\t1000000\t12345\thttps://smartinsidercompanypage\tConsumer Staples\tPersonal Care, Drug and Grocery Stores\tPersonal Care, Drug and Grocery Stores\tPersonal Products\t12345678\tSome Company Corp\tSome Company C\t\t\t\t\tCom\tUS\tSCC\t\t\t\t\t\t\t\t-999\tSome new unexpected event.\t\t\t\t\t\t\t\t2020-07-27\t\t\t\t2020-07-27  13:57:37\tUS\thttps://smartinsiderdatapage\tOn Market\tIssuer\tNot Reported\t\t\t\t\t"; 
+            var realRawIntentionLine = "\"BI12345\"\t\"Some new event\"\t2020-07-27\t2009-11-11\t\"US1234567890\"\t\"\"\t12345\t\"https://smartinsidercompanypage.com\"\t\"Consumer Staples\"\t" +
+                                       "\"Personal Care, Drug and Grocery Stores\"\t\"Personal Care, Drug and Grocery Stores\"\t\"Personal Products\"\t12345678\t\"Some Company Corp\"\t\"Some-Comapny C\"\t" +
+                                       "\"\"\t\"\"\t\"\"\t\"\"\t\"Com\"\t\"US\"\t\"SCC\"\t\"\"\t\"\"\t\"\"\t\"\"\t\"\"\t\"\"\t\"\"\t\t\t-999\t\"Some unexpected event.\"\t\"\"\t\"\"\t\"\"\t\"\"\t\"\"\t\"\"\t\"" +
+                                       "\"\t2020-07-27\t\"\"\t\"\"\t\"\"\t2020-07-27  13:57:37\t\"US\"\t\"https://smartinsiderdatapage.com\"\t\"On Market\"\t\"Issuer\"\t\"Not Reported\"\t\"\"\t\"\"\t\t" +
+                                       "\"\"\t\t\t\"\"\t\t\t\"\""; 
+
+            var tsv = realRawIntentionLine.Split('\t')
+                .Take(60)
+                .Select(x => x.Replace("\"", ""))
+                .ToList();
+
+            // Remove in descending order to maintain index order
+            // while we delete lower indexed values
+            tsv.RemoveAt(46); // ShowOriginal
+            tsv.RemoveAt(36); // PreviousClosePrice
+            tsv.RemoveAt(14); // ShortCompanyName
+            tsv.RemoveAt(7);  // CompanyPageURL
+
+            var filteredRawIntentionLine = string.Join("\t", tsv);
```

Comment:
Seems we should push all this logic into `SmartInsiderIntention.FromRawData()`/make it shared with the processor